Modules/Daq/include/Daq/DaqTask.h

```diff
@@ -17,18 +17,15 @@
 #define QC_MODULE_DAQ_DAQTASK_H
 
 #include "QualityControl/TaskInterface.h"
+#include <Headers/DAQID.h>
+#include <map>
 
 class TH1F;
-class TGraph;
-class TObjString;
-class TCanvas;
-class TPaveText;
 
 using namespace o2::quality_control::core;
```

Comment:
We shouldn't actually use `using namespace` in headers, but this is prevalent among Tasks in modules. However, it is not a big deal, because we load the classes with ROOT and these headers do not pollute anything.

Reply:
you are abolutely correct. I have removed it because we should try to be consistent.